.github/workflows/publish-command.yml

```diff
@@ -45,8 +49,92 @@ jobs:
             ${{ secrets.DAVINCHIA_PAT }}
   ## Gradle Build
   # In case of self-hosted EC2 errors, remove this block.
-  start-publish-image-runner:
-    name: Start Build EC2 Runner
+  start-publish-image-runner-0:
+    name: Start Build EC2 Runner 0
+    runs-on: ubuntu-latest
+    needs: find_valid_pat
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.inputs.repo }}
+          ref: ${{ github.event.inputs.gitref }}
+      - name: Start AWS Runner
+        id: start-ec2-runner
+        uses: ./.github/actions/start-aws-runner
+        with:
+          aws-access-key-id: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
+          github-token: ${{ needs.find_valid_pat.outputs.pat }}
+  start-publish-image-runner-1:
```

Comment:
I couldn't find a better way of doing these than one-by-one due to how the start-aws-runner and outputs work